CONTRIBUTING.md

```diff
@@ -0,0 +1,143 @@
+# Contributing
+
+Thank you for considering a contribution to Archivematica!
+This document outlines the change submission process for Archivematica, along with our standards for new code contributions.
+Following these guidelines helps us assess your changes faster and makes it easier for us to merge you submission!
```

Comment:
Good catch! Unfortunately I copied this section to several other repo's contributing.mds and now they need typo fixes...